Electron.NET Core 0.3.1 fail to compile base Blazor Web project

### Description

After upgrading **Electron.NET Core** from **0.3.0** to **0.3.1**, the build fails with an exception related to missing metadata for generated static assets.

### Environment

* **Electron.NET Core**: 0.3.1
* **.NET SDK**: 10
* **Node.js**: 24.12.0
* **Target Runtime**: win-x64

### Error Message

```
No metadata 'SourceId' was present for item 'Components\Layout\ReconnectModal.razor.js' and no default value was provided.
```

I have observed the same issue in another project, where the error referenced different generated assets, such as:

* `*.razor.js`
* `app.css`

### Expected Behavior

The project should build successfully after upgrading Electron.NET Core from 0.3.0 to 0.3.1.

### Actual Behavior

The build fails with a `SourceId` metadata exception related to generated static web assets.

### Steps to Reproduce

1. Create a new **Blazor Web** project.
2. Add **Electron.NET**.
3. Build the project.

### Additional Notes

I've also saw that when re-opening the project the file affected by this exception can change, for example now it's "Components\Layout\MainLayout.razor.rz.scp.css", no idea how it calculates this static assets

[electron-sample.zip](https://github.com/user-attachments/files/24217021/electron-sample.zip)

The mentioned fix does indeed fix the build of my project but upon publishing I encounter this error:

```
C:\Users\runneradmin\.nuget\packages\electronnet.core\0.3.1\build\ElectronNET.LateImport.targets(430,7): error MSB4184: The expression "[System.IO.Path]::GetFullPath('')" cannot be evaluated. The path is empty. (Parameter 'path') [<redacted>.csproj]
```

Command to publish:
`dotnet publish -p:PublishProfile=win-x64`

Publish profile:
```
<?xml version="1.0" encoding="utf-8"?>
<Project>
	<PropertyGroup>
		<Configuration>Release</Configuration>
		<Platform>Any CPU</Platform>
		<PublishDir>publish\$(Configuration)\$(TargetFramework)\$(RuntimeIdentifier)\</PublishDir>
		<PublishProtocol>FileSystem</PublishProtocol>
		<TargetFramework>net10.0</TargetFramework>
		<RuntimeIdentifier>win-x64</RuntimeIdentifier>
		<SelfContained>true</SelfContained>
		<PublishSingleFile>false</PublishSingleFile>
		<WebPublishMethod>FileSystem</WebPublishMethod>
	</PropertyGroup>
</Project>
```

--- a/src/ElectronNET/build/ElectronNET.Core.targets
+++ b/src/ElectronNET/build/ElectronNET.Core.targets
@@ -13,7 +13,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <PackageId>$(ElectronPackageId)</PackageId>
+    <PackageId Condition="'$(PackageId)' == ''">$(ElectronPackageId)</PackageId>
   </PropertyGroup>
 
   <ItemGroup>

